src/quicktype-core/language/Java.ts

```diff
@@ -303,7 +309,7 @@ export class JavaRenderer extends ConvenienceRenderer {
     }
 
     protected emitPackageAndImports(imports: string[]): void {
-        const allImports = ["java.util.Map"].concat(this._options.justTypes ? [] : imports);
+        const allImports = ["java.util.*"].concat(this._options.justTypes ? [] : imports);
```

Comment:
Is this because we might need `List`, too?  Is this a standard thing to do in Java?

Reply:
Just to add: another reason why the wildcard wasn't much of a concern to me is because there have already been wildcard imports, e.g. `com.fasterxml.jackson.annotation.*` in the code. So if its the wildcard which you would like being avoided, just let me know. But getting rid of wildcard imports as a whole would then probably be a different issue.